gull_tracking/gull_tracking.ipynb

```diff
@@ -37,13 +35,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hv.extension('bokeh')\n",
-    "\n",
-    "map_tiles  = EsriImagery().opts(alpha=1.0, width=800, height=800, bgcolor='black')\n",
-    "points     = hv.Points(df, ['longitude', 'latitude'])\n",
-    "rasterized = shade(rasterize(points, x_sampling=1, y_sampling=1, width=800, height=800), cmap=fire)\n",
-    "\n",
-    "map_tiles * rasterized"
+    "df.hvplot.points(x='longitude', y='latitude', rasterize=True, tiles='EsriImagery',\n",
```

Comment:
Yeah, I figured it out. The difference was in setting the `cnorm` parameter to 'eq_hist'. It wasn't set to that before.

Reply:
Yep, `eq_hist` is pretty important! It's not the default in Bokeh because that would be surprising for most data, but for Datashader output it's generally the best initial approach, switching to another simpler type of normalization only once you've seen what the data looks like and are ready to focus on a well-behaved subset of it.